### PR TITLE
adds fleshed out queue scenarios

### PR DESCRIPTION
**Pull request by amprokop**
Adds more complete queue fake data.

Connects https://github.com/department-of-veterans-affairs/caseflow/issues/4421

### CODE REVIEW

File: lib/fakes/data/appeal_data.rb [hunk 1]
```diff
@@ -95,5 +95,259 @@ def self.default_records
...
+            codes: %w[02 15 03 5252],
+            labels: ["Compensation", "Service connection", "All Others", "Thigh, limitation of flexion of"] },
+          { vacols_sequence_id: 12,
+            codes: %w[02 15 03 5252],
+            labels: ["Compensation", "Service connection", "All Others", "Thigh, limitation of flexion of"] }
+        ]
+      )
+    ]
+    records.map(&:save)
+    records
```
Comment (dannysellers) [inline lib/fakes/data/appeal_data.rb]: Nit: After we screenshared it occurred to me that we could do the same thing by appending `.each(&:save)` to the end of the list

File: lib/fakes/queue_repository.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class << self
   end
 
   def self.tasks_for_user(_css_id)
-    appeal_records || Fakes::Data::AppealData.default_records.map do |record|
+    appeal_records || Fakes::Data::AppealData.default_queue_records.map do |record|
       # For now, we're using the default appeal records, which
```
Comment (dannysellers) [inline lib/fakes/queue_repository.rb]: Nit: Does this comment still apply?

File: lib/fakes/hearing_repository.rb [hunk 3]
```diff
@@ -71,6 +71,13 @@ def self.create_hearing_for_appeal(i, appeal)
     Generators::Hearing.create(random_attrs(i).merge(user: user, appeal: appeal))
   end
 
+  def self.create_already_held_hearing_for_appeal(i, appeal)
```
Comment (amprokop) [inline lib/fakes/hearing_repository.rb]: fyi @sharonwarner, adding this little method here

Comment (amprokop) [reply lib/fakes/hearing_repository.rb]: feel free to modify it any way you like if it doesn't make sense as long as the return value remains the same

**Pull request merged**